idaes/models_extra/column_models/util.py

```diff
@@ -331,3 +296,79 @@ def rule_enth(model, t):
                 ref = Reference(var)
                 setattr(model, "_" + k + "_" + port.local_name + "_ref", ref)
                 port.add(ref, k)
+
+
+# Rule for mole fraction
+def _rule_mole_frac_0(model, t, i, local_name_frac, local_name_flow, phase):
+    sum_flow_comp = sum(
+        model.properties_out[t].component(local_name_frac)[p, i]
+        * model.properties_out[t].component(local_name_flow)[p]
+        for p in phase
+    )
+
+    return sum_flow_comp / sum(
+        model.properties_out[t].component(local_name_flow)[p] for p in phase
+    )
+
+
+# Rule for mole fraction
+def _rule_mole_frac_1(model, t, i, local_name_flow, phase):
+    sum_flow_comp = sum(
+        model.properties_out[t].component(local_name_flow)[p, i] for p in phase
+    )
+
+    return sum_flow_comp / sum(
+        model.properties_out[t].component(local_name_flow)[p, i]
+        for p in phase
+        for i in model.config.property_package.component_list
+    )
+
+
+def _rule_flow_0(model, t, local_name, side_sf):
+    return (model.properties_out[t].component(local_name)) * (side_sf)
+
+
+# Rule to link the flow to the port
+def _rule_flow_1(model, t, i, local_name, side_sf):
+    return (model.properties_out[t].component(local_name)[i]) * (side_sf)
+
+
+# Rule to link the flow to the port
+def _rule_flow_2(model, t, local_name_flow, side_sf, phase):
+    return sum(model.properties_out[t].component(local_name_flow)[p] for p in phase) * (
+        side_sf
+    )
+
+
+# Rule to link the flow to the port
+def _rule_flow_3(model, t, i, local_name_flow, side_sf, phase):
+    return sum(
+        model.properties_out[t].component(local_name_flow)[p, i] for p in phase
+    ) * (side_sf)
+
+
+def _rule_flow_4(model, t, p, i, local_name, side_sf, phase, equipmentType):
+    # If statement to skip in case equipment is not a Tray
+    if equipmentType is None:
```

Comment:
```suggestion
def _rule_flow_4(model, t, p, i, local_name, side_sf, phase, equipment_type):
    # If statement to skip in case equipment is not a Tray
    if equipment_type is None:
```

Reply:
Nevermind, I see that the non-`snake_case` capitalization is present in the top-level function, and changing it would likely affect backward compatibility.